modules/storages/spec/features/delete_project_storage_and_file_links_spec.rb

```diff
@@ -33,9 +33,9 @@
 
 # Test if the deletion of a ProjectStorage actually deletes related FileLink
 # objects.
-RSpec.describe 'Delete ProjectStorage with FileLinks', js: true, webmock: true do
+RSpec.describe 'Delete ProjectStorage with FileLinks', :js, :webmock do
   let(:user) { create(:user) }
-  let(:role) { create(:existing_role, permissions: [:manage_storages_in_project]) }
+  let(:role) { create(:project_role, permissions: [:manage_storages_in_project]) }
```

Comment:
Factory was renamed from `:existing_role` to `:existing_project_role`. If you found the `:existing_role` behavior to be superfluous here, please disregard.

```suggestion
  let(:role) { create(:existing_project_role, permissions: [:manage_storages_in_project]) }
```